src/api-explorer/v4-0/ConcurRequest.swagger2.json

```diff
@@ -0,0 +1,4621 @@
+            }
+          },
+          "400": {
+            "description": "The request is invalid",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "401": {
+            "description": "Request is not authenticated"
+          },
+          "403": {
+            "description": "Principal is not allowed to perform operation"
+          },
+          "500": {
+            "description": "An unexpected error has prevented the operation",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "503": {
+            "description": "The service is currently unavailable",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/travelagencies/{agencyUuid}": {
+      "get": {
+        "tags": [
+          "Travel Agency Resource"
+        ],
+        "summary": "Get the description of a travel agency office",
+        "operationId": "getUsingGET_7",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "agencyUuid",
+            "in": "path",
+            "description": "The unique identifier of the Travel Agency",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ResourceOfTravelAgency"
+            }
+          },
+          "400": {
+            "description": "The request is invalid",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "401": {
+            "description": "Request is not authenticated"
+          },
+          "403": {
+            "description": "Principal is not allowed to perform operation"
+          },
+          "500": {
+            "description": "An unexpected error has prevented the operation",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "503": {
+            "description": "The service is currently unavailable",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ActionOfImportItinProposalAction": {
+      "type": "object",
+      "properties": {
+        "emtAction": {
+          "$ref": "#/definitions/ImportItinProposalAction"
+        }
+      }
+    },
+    "AgencyProposal": {
+      "type": "object",
+      "required": [
+        "agencyProposalEntries",
+        "booked",
+        "importDate"
+      ],
+      "properties": {
+        "agencyProposalEntries": {
+          "type": "array",
+          "description": "List of the proposal entries",
```

Comment:
List of the Agency Proposals ? to check with Charlotte

Reply:
Where is this property field? Agency proposals are not yet externalised so I don't know if this is part of the field available in https://developer.concur.com/api-reference/request/v4.endpoints.html
If not available in the public endpoints resources, please remove